flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -30,7 +31,7 @@ class TiledTileSet
 	
 	public var tileImagesSources:Array<TiledImageTile>;
 	
-	public function new(data:Dynamic)
+	public function new(data:Dynamic, rootPath:String="")
```

Comment:
Hm, we should really use one of the `OneOf` types to clearly define the acceptable types here like we do for `FlxGraphicAsset` etc..